Not able to `go get`

Hi There,

I am facing some trouble during installation.  Could you please guide me what I might be doing wrong here?

`❯ go get -f -u -v  github.com/nohupped/ADtoLDAP`

```
github.com/nohupped/ADtoLDAP (download)
github.com/nohupped/glog (download)
gopkg.in/ini.v1 (download)
gopkg.in/ldap.v2 (download)
gopkg.in/asn1-ber.v1 (download)
github.com/nohupped/ADtoLDAP/gosyncmodules
# github.com/nohupped/ADtoLDAP/gosyncmodules
src/github.com/nohupped/ADtoLDAP/gosyncmodules/logger.go:29:20: cannot use glog.DebugLevel (type uint) as type *uint in argument to logger.SetLogLevel
src/github.com/nohupped/ADtoLDAP/gosyncmodules/logger.go:36:21: cannot use glog.ErrorLevel (type uint) as type *uint in argument to logger.SetLogLevel
src/github.com/nohupped/ADtoLDAP/gosyncmodules/logger.go:39:21: cannot use glog.WarnLevel (type uint) as type *uint in argument to logger.SetLogLevel
src/github.com/nohupped/ADtoLDAP/gosyncmodules/logger.go:42:21: cannot use glog.InfoLevel (type uint) as type *uint in argument to logger.SetLogLevel
src/github.com/nohupped/ADtoLDAP/gosyncmodules/logger.go:45:21: cannot use glog.DebugLevel (type uint) as type *uint in argument to logger.SetLogLevel
```

Thanks

--- a/go.mod
+++ b/go.mod
@@ -3,9 +3,11 @@ module ADtoLDAP
 go 1.13
 
 require (
+	github.com/nohupped/ADtoLDAP v0.0.0-20200426131947-c166814580d5
 	github.com/nohupped/glog v0.0.0-20200104121806-ca610645981a
 	github.com/smartystreets/goconvey v1.6.4 // indirect
 	gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d // indirect
 	gopkg.in/ini.v1 v1.55.0
+	github.com/nohupped/ADtoLDAP v0.0.0-20200426131947-c166814580d5
 	gopkg.in/ldap.v2 v2.5.1
 )

--- a/main.go
+++ b/main.go
@@ -2,7 +2,7 @@ package main
 
 import (
 	"os/user"
-	"ADtoLDAP/syncer"
+	"github.com/nohupped/ADtoLDAP/syncer"
 	"os"
 	"reflect"
 	"flag"

